src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -1,42 +1,53 @@
 import clsx from 'clsx';
 import debounce from 'lodash.debounce';
-import React, { type ReactNode } from 'react';
+import React, { createContext, useContext, type ReactNode } from 'react';
 import { flattenReactChildren } from '../../util/flattenReactChildren';
 import BreadcrumbsItem from '../BreadcrumbsItem';
 import Menu from '../Menu';
 import styles from './Breadcrumbs.module.css';
 
 type Props = {
+  /**
+   * aria-label for `nav` element to describe Breadcrumbs navigation to screen readers
+   */
+  'aria-label'?: string;
   /**
    * Child node(s) that can be nested inside component
    */
   children?: ReactNode;
   /**
-   * CSS class names that can be appended to the component.
+   * CSS class names that can be appended to the component
    */
   className?: string;
-  /**
-   * aria-label for `nav` element to describe Breadcrumbs navigation to screen readers
-   */
-  'aria-label'?: string;
-
   /**
    * HTML id for the component
    */
   id?: string;
+  /**
+   * Custom string separator between individual breadcrumbs
+   * Defaults to '/'
+   */
+  separator?: string;
```

Comment:
Hesitant to allow separators other than strings right now

Reply:
Fair. In fact, I think we can make separator a subset of possible strings: 

* "|"
* "/"
* ">"

Specifying exact string values lets us also constrain the length without having to write `.length` checks in code.

In the future, also maybe icon? can cross that bridge when we get to it. This would be a good followup with design... figma lets you create lists in the UI that could let designers select from a set of values too, so good parity

Reply:
I'm down to make it a subset of a few possible strings and not having icon for now. Will follow up with design

Reply:
Followed up with design, this subset looks good